webex_assistant_sdk/app.py

```diff
@@ -17,6 +19,7 @@ def __init__(
         super().__init__(import_name, responder_class=responder_class, **kwargs)
         self.secret = secret
         self.private_key = private_key
+        self._use_encryption = not os.environ.get('WXA_SKILL_DEBUG', False)
```

Comment:
Why do we do this here when it can be just directly done in `webex_assistant_sdk/server.py`? It seems redundant to initialize a variable here and just pass it to another component instead of just initializing it where its needed.

Reply:
Done